certified-connectors/Revizto (Ireland)/README.md

```diff
@@ -0,0 +1,43 @@
+# Revizto (Ireland)
```

Comment:
please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md). All of the required headings should be present.